TwitterPolling/poll-trends.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+
+#-----------------------------------------------------------------------
+# twitter-trends
+#  - lists the current global trending topics
+#-----------------------------------------------------------------------
+
+from twitter import *
+
+#-----------------------------------------------------------------------
+# load our API credentials
+#-----------------------------------------------------------------------
+import sys
+sys.path.append(".")
+import config
+
+#-----------------------------------------------------------------------
+# create twitter API object
+#-----------------------------------------------------------------------
+twitter = Twitter(auth = OAuth(config.access_key,
+                  config.access_secret,
+                  config.consumer_key,
+                  config.consumer_secret))
+
+#-----------------------------------------------------------------------
+# retrieve global trends.
+# other localised trends can be specified by looking up WOE IDs:
+#   http://developer.yahoo.com/geo/geoplanet/
+# twitter API docs: https://dev.twitter.com/rest/reference/get/trends/place
+#-----------------------------------------------------------------------
+ny_results = twitter.trends.place(_id = 2459115)
```

Comment:
Constants for these magic numbers would be nice. A good way to set them up might be a config map, i.e.
```
ids = {NY_ID: 2459115, ...}

ny_results = twitter.trends.place(_id = ids.NY_ID)
```